src/components/common/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,67 @@
+import { MdSearch } from 'react-icons/md';
+
+import IconWrapper from '../IconWrapper';
+import Input from '../Input';
+import { SearchBarWrapper } from './SearchBar.style';
+import { SearchBarProps } from './SearchBar.type';
+
+/**
+ *
+ * @description SearchBar 컴포넌트
+ * @summary 사용법 :
+ * import { useForm } from 'react-hook-form';
+
+import SearchBar from '@/components/common/SearchBar';
+
+const TestPage = () => {
+  const { handleSubmit } = useForm();
+
+  const onSubmit = () => {
+    alert('sumbit!');
+  };
+
+  return (
+    <>
+      <SearchBar
+        handleSearchIcon={() => alert('검색됨!')}
+        handleSubmit={handleSubmit(onSubmit)}
+      />
+    </>
+  );
+};
+
+export default TestPage;
+
+ * @param handleSearchIcon : 선택 | 함수. 검색 아이콘에 들어갈 클릭 이벤트를 받음.
+ * @param handleSubmit : 선택 | 함수. input에 검색어를 입력 후 엔터 할 때 submit 이벤트를 받음
+ * @param ...props : 커스텀을 위함 (+ react-hook-form)
+ * @returns
+ */
+
+const SearchBar = ({
+  handleSearchIcon,
+  handleSubmit,
+  ...props
+}: SearchBarProps) => {
+  return (
+    <SearchBarWrapper>
+      <form onSubmit={handleSubmit}>
+        <Input
+          width={'100%'}
+          margin={0}
+          placeholder={'검색어를 입력해주세요.'}
+          style={{ paddingRight: '3rem' }}
```

Comment:
P5; `style={{ paddingRight: '3rem' }}`  이 코드는 무슨역할인지 궁금하여서 여쭈어봅니다 !

Reply:
style={{ paddingLeft: '1rem' }} 도 좋아보입니다 너무 앞에있는 느낌이라...디자인 어렵네요 ㅠㅠ

Reply:
P5; 저도 왼쪽여백이 있으면 좋을 것 같다는 의견에 한 표 던집니다..!!
```suggestion
         style={{
            paddingLeft: '.5rem',
            paddingRight: '3rem',
          }}
```

Reply:
> P5; style={{ paddingRight: '3rem' }} 이 코드를 적용했을 때 아무일도 안일어나는거 같습니다..?!

padding-right 값을 준 이유는 `input의 내용과 검색 아이콘이 겹쳐지는 것을 막기 위함` 입니다 !!

padding-right 값이 없다면 아래 사진과 같아집니다 !
<img width="438" alt="스크린샷 2024-02-20 오후 6 01 14" src="https://github.com/Team-kiwing/Team-3seco-kiwing-fe/assets/97944429/a33a6c41-501c-4c5a-a772-38ceb5cd522f">

padding-right 값이 있다면 아래 사진과 같습니다 !
<img width="438" alt="스크린샷 2024-02-20 오후 6 00 56" src="https://github.com/Team-kiwing/Team-3seco-kiwing-fe/assets/97944429/b7194858-3ed7-47f3-a06d-cfb273b42572">

헷갈릴 수 있는 부분은 PR에 남겼어야 했는데 제가 뺴먹은 것 같아요 ㅠㅠ 혼란을 드려 죄송합니다 !!

> P5; style={{ paddingLeft: '1rem' }} 도 좋아보입니다 너무 앞에있는 느낌이라...디자인 어렵네요 ㅠㅠ

수정하겠습니다 !!